packages/shared/clickhouse/migrations/clustered/0030_add_eval_execution_trace_id_to_scores.down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE scores ON CLUSTER '{cluster}' DROP COLUMN IF EXISTS execution_trace_id;
```

Comment:
I think you'll have to use cluster `default` here. Afaik there is no build in templating around our executions.